guides/v2.2/cloud/project/log-locations.md

```diff
@@ -0,0 +1,148 @@
+-  **Support report log**: `var/log/support_report.log`
+
+Logs from the deploy hook are unique for each environment. The deploy log is in the following directories:
+
+-  **Starter and Pro Integration**: `/var/log/deploy.log`
+-  **Pro Staging**: `/var/log/platform/<project_id>_stg/deploy.log`
+-  **Pro Production**: `/var/log/platform/<project_id>/deploy.log`
+
+The value of `<project_id>` depends on the project ID and whether the environment is Staging or Production. For example, with a project ID of `yw1unoukjcawe`, the Staging environment user is `yw1unoukjcawe_stg` and the Production environment user is `yw1unoukjcawe`.
+Using that example, the deploy log is: `/var/log/platform/yw1unoukjcawe_stg/deploy.log`
+
+The log for each deployment concatenates to the specific `deploy.log` file.
+
+#### To view the deploy log:
+
+```bash
+magento-cloud log deploy
+```
+
+Sample response:
+
+```terminal
+Reading log file project-integration-ID--mymagento@ssh.magento.cloud:/var/log/'deploy.log'
+================
+
+Error Output:
+================
+sh: 1: kill: No such process
+
+[2019-09-09 09:00:00] NOTICE: Validating configuration
+...
+```
+
+#### To view a deploy log from the Staging environment:
+
+```bash
+magento-cloud log platform/<project_id>_stg/deploy
+```
+
+## Application logs
+
+Similar to deploy logs, application logs are unique for each environment. The following table lists application log locations based on their environment:
+
+Log file            | Starter and Pro Integration | Pro Staging                                       | Pro Production
+------------------- | --------------------------- | ------------------------------------------------- | -------------------------------------------
+**Deploy log**  | `/var/log/deploy.log`       | `/var/log/platform/<project_id>_stg/deploy.log`   | `/var/log/platform/<project_id>/deploy.log`
+**Cron log**        | `/var/log/cron.log`         | `/var/log/platform/<project_id>_stg/cron.log`     | `var/log/platform/<project_id>/cron.log`
+**Nginx access log**| `/var/log/access.log`       | `/var/log/platform/<project_id>_stg/access.log`   | `/var/log/platform/<project_id>/access.log`
+**Nginx error log** | `/var/log/error.log`        | `/var/log/platform/<project_id>_stg/error.log`    | `/var/log/platform/<project_id>/error.log`
+**PHP access log**  | `/var/log/php.access.log`   | `/var/log/platform/<project_id>_stg/php.access.log` | `/var/log/platform/<project_id>/php.access.log`
+**PHP FPM log**     | `/var/log/app.log`          | `/var/log/platform/<project_id>_stg/php5-fpm.log` | `/var/log/platform/<project_id>/php5-fpm.log`
+
+## Service logs
+
+Because each service runs in a separate container, the service logs are not available in the Integration environment. {{site.data.var.ece}} only provides access to the web server container in the Integration environment. The following service log locations are for the Pro Staging and Production environments:
```

Comment:
```suggestion
Because each service runs in a separate container, the service logs are not available in the Integration environment. {{site.data.var.ece}} provides access to the web server container in the Integration environment only. The following service log locations are for the Pro Staging and Production environments:
```